packages/web/components/place-limit-tool/index.tsx

```diff
@@ -129,7 +129,11 @@ export const PlaceLimitTool: FunctionComponent<PlaceLimitToolProps> = observer(
               <span className="body2 text-osmoverse-300">
                 {swapState.baseAsset?.coinDenom}{" "}
                 {t("assets.table.price").toLowerCase()} ≈{" "}
-                {formatPretty(swapState.priceState.spotPrice ?? new Dec(0))}{" "}
+                {formatPretty(
+                  orderDirection === "bid"
```

Comment:
I might be wrong here, so you want to display the ask spot price when the direction is bid?

Reply:
A market order is run against the price of the opposite direction